tests/managers/index.js

```diff
@@ -26,6 +28,9 @@ describe('ManagerTests', function () {
                 fileManagerTests(appClient);
                 taskTests(appClient);
                 queueTests(appClient);
+                voiceCommandUpdateOperationTests(appClient);
```

Comment:
The VoiceCommandManagerTests are failing for me. If I do them before all the other tests though, they pass. It seems that they are being run after dispose() is called and as the manager is in the SHUTDOWN state.